lib/customer.rb

```diff
@@ -0,0 +1,36 @@
+class Customer
+  attr_accessor :name, :phone, :my_order
+  
+  def initialize(args)
+    self.name = args[:name]
+    self.phone = args[:phone]
+    self.my_order = []
+  end
+  
+  def order_dish(restaurant, dish, qty)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `restaurant`. If it's necessary, use `_` or `_restaurant` as an argument name to indicate that it won't be used.